CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * Fix a crash caused when editing string option (#4852, @rbdixon)
 * Base container image bumped to Debian 11 Bullseye (@Kriechi)
 * Upstream replays don't do CONNECT on plaintext HTTP requests (#4876, @HoffmannP)
+* Remove old pyOpenSSL cruft (#4831, @KarlParkinson)
```

Comment:
Let's call this "remove workarounds for old pyOpenSSL versions", in retrospect I realize that my wording from the issue could be misunderstood. pyOpenSSL is not crufty. :)